Gemfile

```diff
@@ -47,6 +48,7 @@ end
 group :test do
   # an XML formatter is required for fancier CircleCI results
   gem 'rspec_junit_formatter'
+  gem 'rails-controller-testing'
```

Comment:
I really appreciate you taking the time to add tests! 🙌🏾

That said, i'm quite hesitant to add this gem since the rails team very intentionally [deprecated and removed](https://github.com/rails/rails/issues/18950) access to `assigns`.

Relatedly, we've been preferring [request specs](https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec) over controller specs. My impression is that is becoming the recommended best practice.

I'd love to hear your thinking on all this?

Reply:
I can see the hesitation towards `assigns` and the `rails-controller-testing` gem. I introduced them because what I care about testing here wasn't so much the view rendering but if the count of `people` is what I'd expect. 

Without the gem I could do the same test with an assertion against `controller.instance_variable_get(:@people)` but I'm not sure that's much better (reaching into the instance variables of an object without a reader is smelly code). 

I could move this to a request spec and assert that the expected number of `<td>`s exist or `<td>`s with the expected people's names in them.